File: C/8 Kyu/calculate_average.c
double find_average(const double array[/* length */], unsigned length)
{
    if(!length){
      return 0;
    }
    double total = 0.0;
	  for(int i = 0; i< length; i++){
      total+=array[i];
    }
  return total/length;
}

File: Rust/8 Kyu/calculate_average.rs
fn find_average(slice: &[f64]) -> f64 {
    let tam = slice.len() as f64;
    if tam ==0.0{
        return 0.0;
    }
    let mut soma:f64 = 0.0;
    for x in slice{
        soma+=x;
        
    }
    soma/tam
}